css/common.css

```diff
@@ -290,6 +290,14 @@ dl.fqa > dd::before {
 	content: 'A.';
 	color: hsl(30,75%,45%);
 }
+dl.fqa > dt:lang(tr)::before {
+	content: 'S.';
+	color: hsl(200,60%,45%);
+}
+dl.fqa > dd:lang(tr)::before {
+	content: 'Y.';
+	color: hsl(30,75%,45%);
+}
```

Comment:
Here (and the same for 'Y.' below), you don't need to set the `color` again. Just setting `content` is enough.